CRM/Volunteer/Upgrader.php

```diff
@@ -116,7 +111,12 @@ public function installProjectRelationships() {
     );
 
     foreach ($options as $opt) {
-      civicrm_api3('OptionValue', 'create', $optionDefaults + $opt);
+      $getOptionValues = civicrm_api3('OptionValue', 'get', $optionDefaults + $opt);
```

Comment:
seems wrong that defaults would not be overwritten, as would be the case if using array_merge instead.